requirements_docs.txt

```diff
@@ -1,6 +1,6 @@
 sphinx>=1.4.3
 sphinxcontrib-napoleon
-sphinx_gallery
+sphinx-gallery
```

Comment:
`pip` does not care about `-` or `_` but conda is unambiguous and accepts only one. This now works with both `conda` and `pip`.